pkg/vault/vault.go

```diff
@@ -78,14 +79,19 @@ func (a App) getVaultKey(filename string, key string, getVaultContent func(strin
 		return strings.Trim(rawVault, "\n"), nil
 	}
 
-	var vaultContent map[string]string
+	var vaultContent = make(map[string]interface{})
 	if err := yaml.Unmarshal([]byte(rawVault), &vaultContent); err != nil {
 		return "", err
 	}
 
 	for vaultKey, vaultValue := range vaultContent {
 		if strings.EqualFold(key, vaultKey) {
-			return strings.Trim(vaultValue, "\n"), nil
+			switch v := vaultValue.(type) {
+			case string:
+				return strings.Trim(v, "\n"), nil
+			case int:
+				return strconv.Itoa(v), nil
```

Comment:
Can you add bool in your switch (part of common yaml type) ?

Reply:
Added.